CONTRIBUTING.md

```diff
@@ -11,18 +11,13 @@ than it was before.
 
 ## Submitting changes
 
-1. Make a PR for every change.
-
-   Even seemingly trivial patches can break things in surprising ways.
-Use of common sense is OK. If you're only fixing a typo in a comment,
-it's probably fine to just push it. But if in doubt, open a PR.
-
-2. Get at least one +1 on your PR before you push.
+1. Get at least one +1 on your PR before you push.
 
    For simple patches, it will only take a minute for someone to review
-it.
+it. PR force-push breaks unadressed review comments, so it is better to
```

Comment:
This is not relevant to the opening sentence of point 1, and doesn't convey the real O(N^2) cost of review :)

Suggestion to reword as:
"2. Don't force push small changes after making the PR ready for review. Doing so will force readers to re-read your entire PR, which will delay the review process."